src/xdocs/config_misc.xml

```diff
@@ -1084,6 +1084,23 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
           times it is practical experience. In any case, this check should
           just ensure that a minimal set of indentation rules is followed.
         </p>
+        <p>
+          Basic offset indentation is used for indentation inside code blocks.
+          Line wrapping indentation is used when statement or expression is placed on
+          few lines.
+        </p>
+        <p>Example:</p>
+        <source>
+if ((condition1 &amp;&amp; condition2)
+        || (condition3 &amp;&amp; condition4)    // line wrap with bigger indentation
+        ||!(condition5 &amp;&amp; condition6)) { // line wrap with bigger indentation
```

Comment:
This line says `line wrap with bigger indentation`. Another one below says `line wrap`? Any reason for the difference?

Reply:
I tried to show example we are ok to indent further to the right to increase read-ability, so it is intended not strictly by linewrap value